content/en/docs/guides/setup/rpi4_setup/index.md

```diff
@@ -75,8 +148,8 @@ In the Raspberry terminal, run:
 curl -fsSL https://get.docker.com -o get-docker.sh
 sudo sh get-docker.sh
 
-# Grant priviledges to the rospi user to run docker containers
-sudo usermod -aG docker raspi
+# Grant priviledges to the Rpi Username  that was seted during the extra setting Raspberry Pi  , to run docker containers
```

Comment:
Typo in seted, should be "set".

Also, extra space before a comma.

Reply:
Solved